src/util/sync_blocks.ts

```diff
@@ -28,28 +46,31 @@ export class SyncBlocks {
 				} else {
 					this.startQueue(conn, chain);
 				}
-			});
-		});
+			})
+			.catch((error: any) => {
+				console.log("\n!!!!\nerror: " + error);
+			});;
+		})
 		this.statsHandler.getStatsInfo().then((stats: IStats) => {
 			if (!stats) {
 				this.statsHandler.create();
 			}
 		});
 	}
 
-	stopSync(blockQueue: BlockQueue) {
-		return blockQueue.stop();
-	}
-
 	initChainInfo(connection: Connection, isUpdate: boolean): Promise<IChain> {
 		return new Promise((resolve: Function, reject: Function) => {
-			connection.getLastBlock().then((block: any) => {
+			connection.getLastBlock()
+			.then((block: any) => {
 				let chain: IChain = { chainId: block.header.chain_id, blockHeight: 0 };
 				if (isUpdate) {
 					resolve(this.chainHandler.update(chain));
 				} else {
 					resolve(this.chainHandler.create(chain));
 				}
+			})
+			.catch(err=>{
```

Comment:
can we please wrap err like this: (err: any) => {} 
(we need to try to keep formatting and code style the same)